pallets/parachain-staking/src/migrations.rs

```diff
@@ -131,13 +131,11 @@ impl<T: Config> OnRuntimeUpgrade for SplitDelegatorStateIntoDelegationScheduledR
 					});
 				}
 
-				let new_state = Delegator {
-					id: old_state.id,
-					delegations: old_state.delegations,
-					total: old_state.total,
-					less_total: old_state.requests.less_total,
-					status: old_state.status,
-				};
+				let mut new_state = Delegator::default_with_total(old_state.id, old_state.total);
```

Comment:
this can probably be reverted now (and `default_with_total` removed) since `total` is still `pub`. I only added this and `default_with_total` since `total` wasn't `pub` in the other branch

Reply:
10abaee

Reply:
Revent :roll_eyes: